move-text.el

```diff
@@ -67,11 +67,15 @@ Note: `region-beginning' and `region-end' are the reason why an
 
 So the predicate `region-active-p' is needed to avoid calling
 them when there's no region."
-    `(,@(if (region-active-p)
-            (list (region-beginning) (region-end))
-          (list nil nil))
-      ,current-prefix-arg))
-
+    ;; `(,@(if (region-active-p)
+    ;;         (list (region-beginning) (region-end))
+    ;;       (list nil nil))
+    ;;   ,current-prefix-arg))
+    (list
```

Comment:
I got keeping errors when I tried C-u M-x move-text-up or down  because the current-prefix-arg needs to be part of the list returned by the function. `(prefix-numeric-value current-prefix-arg)` simplifies the functions, because you always have an integer there